cg/meta/backup/pdc.py

```diff
@@ -39,14 +40,17 @@ def validate_is_dsmc_running(cls) -> bool:
             Exception: for all non-exit exceptions.
```

Comment:
Minor observation: If the function does not raise any error then it should not be called validate, just `is_dsmc_running`, but feel free to disregard

Reply:
We do not explicitly raise in cg but psutil will on error and we catch the exceptions. Seems like a special case - will leave it for now